Lib/http/server.py

```diff
@@ -689,6 +689,7 @@ def send_head(self):
                              parts[3], parts[4])
                 new_url = urllib.parse.urlunsplit(new_parts)
                 self.send_header("Location", new_url)
+                self.send_header("Content-Length", "0")
```

Comment:
Please add a test case for this change.

Reply:
I've added a single check to one of the existing http server tests.

If something else is warranted in this case, I'm more than happy to do it, but I'd need some direction.
The added check seems sufficient to me and in-keeping with the way those existing tests are written.